lib/matplotlib/pyplot.py

```diff
@@ -2690,7 +2690,13 @@ def matshow(A: ArrayLike, fignum: None | int = None, **kwargs) -> AxesImage:
     return im
 
 
-def polar(*args, **kwargs) -> list[Line2D]:
+def polar(
+        *args: float | ArrayLike | str,
+        scalex: bool = True,
+        scaley: bool = True,
+        data=None,
```

Comment:
IMHO we could leave data out, because it's not typed anyway, so no advantage of writing it out.